components/collective-navbar/index.js

```diff
@@ -258,7 +258,9 @@ const getDefaultCallsToActions = (collective, sections, isAdmin, isHostAdmin, Lo
     hasManageSubscriptions: isAdmin && get(features, 'RECURRING_CONTRIBUTIONS') === 'ACTIVE',
     hasDashboard: isAdmin && isFeatureAvailable(collective, 'HOST_DASHBOARD'),
     hasRequestGrant:
-      (isFund || get(settings, 'fundingRequest') === true) && expenseSubmissionAllowed(collective, LoggedInUser),
+      get(settings, 'expenseTypes.hasGrant') === true &&
```

Comment:
My previous feedback on `isFund` was related to your original specs, where grants were supposed to be turned off by default. With the feedback received, are you sure that we don't want this `isFund ||`?

Reply:
@Betree : Yes, sorry, I missed that; nowt this I think is a bit more complicated than adding just `isFund` back. We also need to make sure the checkbox toggle we added is in sync with both the settings, `expenseTypes.hasGrant` as well as `disableGrantsByDefault`. I've made the necessary changes and updated the PR with a screen-cast. Let me know what you think. 🙂